### ISSUE TITLE
[accessibility] Attribution is not accessible using a keyboard

### ISSUE THREAD

**Issue by Malvoz** [route360/Leaflet.CondensedAttribution#3]
The condensed attribution should be a control to toggle the attribution, so that it can be accessed via keyboard.

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
